### PR TITLE
Reference guide

### PR DESCRIPTION
**Pull request by jschueller**
This adds some content to the reference guide (3 methods: kolmogorov test, Monte carlo simulation, sobol indices).

Also the weibull variance was missing some parenthesis.

### CODE REVIEW

File: python/doc/theory/sensitivity_analysis/sensitivity_analysis.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+===========================
+Ranking uncertainty sources
+===========================
+
+Snecitivity analysis methods can be used to analyse the respective importance of each
```
Comment (dbarbier) [inline python/doc/theory/sensitivity_analysis/sensitivity_analysis.rst]: Typo: Snecitivity  -> Sensitivity

File: python/doc/theory/sensitivity_analysis/sensitivity_sobol.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+and :math:`(x_{11}^{(2)},\ldots,x_{1n_X})^{(2)},\ldots,(x_{N^1}^{(2)},\ldots,x_{Nn_X}^{(2)})`
+
+The estimation of sensivity indices for first order consists in estimating the quantity
+
+.. math::
+
+    V_i = \Var{\Expect{ Y^k \vert X_i}} = \Expect{ \Expect{Y^k \vert X_i}^2}  - \Expect{\Expect{ Y^k \vert X_i }} ^2 = U_i - \Expect{Y^k} ^2
+
+Sobol proposes to estimate the quantity :math:`U_i = \Expect{\Expect{ Y^k \vert X_i}^2}`
+by swaping every variables in the two samples except the variable :math:`X_i` between the two calls of the function:
```
Comment (dbarbier) [inline python/doc/theory/sensitivity_analysis/sensitivity_sobol.rst]: Typo: swaping -> swapping

File: python/doc/theory/uncertainty_propagation/monte_carlo_simulation.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+.. math::
+
+    \lim_{N\rightarrow\infty}\Prob{P_f\in[\widehat{P}_{f,\inf},\widehat{P}_{f,\sup}]}=\alpha
+
+with :math:`\widehat{P}_{f,\inf}=\widehat{P}_f - q_{\alpha}\sqrt{\frac{\widehat{P}_f(1-\widehat{P}_f)}{N}}$, $\widehat{P}_{f,\sup}=\widehat{P}_f + q_{\alpha}\sqrt{\frac{\widehat{P}_f(1-\widehat{P}_f)}{N}}`
+and :math:`q_\alpha` is the :math:`(1+\alpha)/2`-quantile of the standard normal distribution.
+
+One can also use a convergence indicator that is independent of the confidence
+level $\alpha$: the coefficient of variation, which is the ratio between the
+asymptotic standard deviation deviation of the estimate and its mean value.
```
Comment (dbarbier) [inline python/doc/theory/uncertainty_propagation/monte_carlo_simulation.rst]: Typo: double word deviation in this sentence

File: python/doc/theory/uncertainty_propagation/monte_carlo_simulation.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+It is a relative measure of dispersion given by:
+
+.. math::
+
+    \textrm{CV}_{\widehat{P}_f}=\sqrt{ \frac{1-\widehat{P}_f}{N \widehat{P}_f}}\simeq\frac{1}{\sqrt{N\widehat{P}_f}}\mbox{ for }\widehat{P}_f\ll 1
+
+It must be emphasize that these results are *asymptotic* and as such needs that :math:`N\gg 1`.
```
Comment (dbarbier) [inline python/doc/theory/uncertainty_propagation/monte_carlo_simulation.rst]: Typo: emphasize -> emphasized

File: python/doc/theory/uncertainty_propagation/uncertainty_propagation.rst [hunk 5]
```diff
@@ -0,0 +1,16 @@
+.. _uncertainty_propagation:
+
+===================================
+Methods for uncertainty propagation
+===================================
+
+This section is organized according to the different uncertainty criterion.
```
Comment (dbarbier) [inline python/doc/theory/uncertainty_propagation/uncertainty_propagation.rst]: Typo: criterion -> criteria

**Comment by jschueller** [comment]
@regislebrun are we good on this one ? it's what I've shown last time

**Comment by regislebrun** [comment]
@jschueller I just recover from another bad sequence with my intestine. I will have a look at this PR asap, I am still working on the upcoming User's Day.

**Comment by jschueller** [comment]
@regislebrun sorry to hear that, take your time then.

**Comment by regislebrun** [comment]
+1

**Pull request merged**